### PR TITLE
✨(i18n) add i18n support

### PR DESCRIPTION
**Pull request by madmatah**
## Purpose

Ashley should be able to support multiple languages.

## Proposal

- [x] Add i18n support in ashley.
- [x] Translate messages with crowdin.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -135,6 +137,38 @@ migrate:  ## run django migration for the ashley project.
...
+i18n-download-and-compile: \
+  crowdin-download \
+  i18n-compile
+.PHONY: i18n-download-and-compile
+
+i18n-generate: ## create the .po files used for i18n
+	@$(COMPOSE_RUN) -w /app/src/ashley ashley python /app/sandbox/manage.py makemessages --keep-pot
+.PHONY: messages
+
+i18n-generate-and-upload: ## generate source translations for all applications and upload then to crowdin
```
Comment (sampaccoud) [inline Makefile]:
```suggestion
i18n-generate-and-upload: ## generate source translations for all applications and upload them to crowdin
```

File: src/ashley/locale/fr_FR/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+#: templates/forum_conversation/topic_list.html:54
+#, python-format
+msgid "By: <a href=\"%(poster_url)s\">%(username)s</a>"
+msgstr "Par : <a href=\"%(poster_url)s\">%(username)s</a>"
+
+#: templates/forum/forum_list.html:81 templates/forum/forum_list.html:154
+#: templates/forum_conversation/topic_list.html:58
+#, python-format
+msgid "By: %(poster_username)s"
+msgstr "Par: %(poster_username)s"
```
Comment (sampaccoud) [inline src/ashley/locale/fr_FR/LC_MESSAGES/django.po]:
```suggestion
msgstr "Par : %(poster_username)s"
```

File: src/ashley/locale/fr_FR/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+msgstr "Vues"
+
+#: templates/forum_conversation/topic_list.html:29
+msgid "This topic is locked"
+msgstr "Cette discussion est verrouillée"
+
+#: templates/forum_conversation/topic_list.html:34
+#, python-format
+msgid "By: <a href=\"%(poster_url)s\">%(username)s</a> on %(creation_date)s"
+msgstr "Par: <a href=\"%(poster_url)s\">%(username)s</a> sur %(creation_date)s"
```
Comment (sampaccoud) [inline src/ashley/locale/fr_FR/LC_MESSAGES/django.po]:
```suggestion
msgstr "Par : <a href=\"%(poster_url)s\">%(username)s</a> sur %(creation_date)s"
```

File: src/ashley/locale/fr_FR/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+
+#: templates/forum_conversation/topic_list.html:38
+#, python-format
+msgid "By: %(poster_username)s on %(creation_date)s"
+msgstr "Par: %(poster_username)s le %(creation_date)s"
```
Comment (sampaccoud) [inline src/ashley/locale/fr_FR/LC_MESSAGES/django.po]:
```suggestion
msgstr "Par : %(poster_username)s le %(creation_date)s"
```

**Comment by madmatah** [comment]
Thanks @sampaccoud , I addressed your feedbacks on #19

**Pull request merged**